CorsixTH/Luatest/spec/entities/humanoids/vip_spec.lua

```diff
@@ -0,0 +1,140 @@
+--[[ Copyright (c) 2020 <tbc>
```

Comment:
I'll suggest we use your name here @sadger as the spec file is your baby. But happy to help as needed on irc.

Reply:
Copy from epidemic.lua

Reply:
Fine by me :+1: